docs/translations/api-docs/use-select/use-select.json

```
@@ -1,6 +1,21 @@
 {
   "hookDescription": "",
-  "parametersDescriptions": {},
+  "parametersDescriptions": {
+    "buttonRef": "The ref of the trigger button element.",
+    "defaultOpen": "If <code>true</code>, the select will be initially open.",
```

```suggestion
    "defaultOpen": "If <code>true</code>, the select will be open initially.",
```

---

Or to be more direct:

```suggestion
    "defaultOpen": "If <code>true</code>, the select will be open by default.",
```